README.adoc

```diff
@@ -297,8 +333,9 @@ The directory structure of the project should now look like this:
 // Running the tests
 // =================================================================================================
 
-[role='command']
-include::{common-includes}/mvnverify.adoc[]
+=== Running the tests
+
+Since you started Open Liberty in development mode at the start of the guide, press the `enter/return` key to run the tests. You will see the following output:
```

Comment:
Change
"You will see the following output:"
To
"You see the following output:"